wrappers/javascript/aries-askar-nodejs/tests/cryptoBox.test.ts

```diff
@@ -1,9 +1,9 @@
 import { CryptoBox, Key, KeyAlgs } from '@hyperledger/aries-askar-shared'
 
-import { setup } from './utils'
-
 describe('CryptoBox', () => {
-  beforeAll(() => setup())
+  beforeAll(() => {
+    require('@hyperledger/aries-askar-nodejs')
```

Comment:
Don't we do this in the `setup.ts`?  It's not needed for the other files right?

Reply:
The setup.ts setup some logger stuff, but we could also move this `require` call to a setup function if that's preferred.